scheduler/server/job_state.go

```diff
@@ -22,10 +26,13 @@ type jobState struct {
 	TimeMarker     time.Time    //when was this job last marked (i.e. for reporting purposes)
 
 	// track tasks by state (completed, running, not started) for priority based scheduling algorithm
-	// TODO remove if we don't use the priority based scheduling algorithm
-	Completed  map[string]*taskState
-	Running    map[string]*taskState
-	NotStarted map[string]*taskState
+	// Completed and Running are only used by priority based scheduling algorithm
```

Comment:
We deleted the priority algo below, so why do we have these fields still?

Reply:
they are used by the new scheduling algorithm - I'll update the comment.